uw-frame-components/portal/search/partials/search.html

```diff
@@ -7,7 +7,7 @@
             aria-label="Search bar enter the app you are looking for"
             focus-me="showSearchFocus"
             ng-show="$storage.typeaheadSearch"
-            typeahead="title as match.title for match in filterMatches | limitTo:10"
+            uib-typeahead="title as match.title for match in filterMatches | limitTo:10"
```

Comment:
As a _general_ rule, I say err on the side of caution, since version numbers are cheap but bugs & deprecations are not.  I.e.: better for Star to update these directives and not have to than for Star to troubleshoot an unexpected bug and uw-frame to have to deprecate a released version.

And along those lines I would just recommend again keeping accurate, semantic versions for snapshots as well as release versions!  That way it's at least possible for a downstream project to be on the cutting edge and ferret out bugs before a deprecation becomes necessary!  _(not that we would want to actually do that with Star at this time)_